local.yml

```diff
@@ -31,8 +31,8 @@ services:
     image: osler_production_postgres
     container_name: postgres
     volumes:
-      - local_postgres_data:/var/lib/postgresql/data
-      - local_postgres_data_backups:/backups
+      - local_postgres_data:/var/lib/postgresql/data:cached
+      - local_postgres_data_backups:/backups:cached
```

Comment:
How often is the backup updated. Does it make sense to cache it?

Reply:
If I'm totally honest, I have no idea. According to that article, the difference should be imperceptible to the user. I haven't really pushed on it that much but after I did this (and restarted Docker) it really improved the performance.

Reply:
I'm going to say to uncache the backups but keep postgresql cached, since the backup isn't read from very often.